### DISCUSSION TITLE
Training on an image set of more than one subject using the new method

### DISCUSSION THREAD

**Discussion by SAC020** [TheLastBen/fast-stable-diffusion#833] [Q&A]
I have an instance image set containing a number of persons in a specific position / stance, and I am trying to train for the position, not the specific persons / faces. Is it **mandatory** to caption with a unique token for each person (face) while using the new method?

Or, as long as I don't plan to "call" any specific subject, but rather the position, I can use a generic unique instance token for all subjects?

I am referring to the statement: _The most important step is to rename the instance pictures of each subject to a unique unknown identifier_

**Comment by TheLastBen** [comment]
if you're not training on specific faces, use the same identifier for all images
